spec/ngx_helper.lua

```diff
@@ -1,9 +1,17 @@
+require('resty.core')
```

Comment:
Why wasn't this required until now?

Reply:
This is not strictly necessary and will work without it because resty.core is required in the busted.lua. But I want this file to be meaningful standalone and for that I think it is necessary to require resty.core because it overrides ngx methods. So if it would be required at some later point it would not pass the verification.